README.md

```diff
@@ -136,10 +176,17 @@ private API and is subject to change between any versions.
 
 #### How do I use it outside of Node.js
 
-It is **only built for Node.js** environment versions ^12.19.0 || >=14.15.0
+It is **only built for Node.js** environment versions >=16.0.0
+
+#### Why is v2.local and v4.local not supported?
+
+Because OpenSSL does not (yet) offer support for `XChaCha20-Poly1305` and `XChaCha20`. Once it does, 
+and is readily available in Node.js, `v2.local` and `v4.local` will be added.
```

Comment:
Does OpenSSL3 v3.x also offer BLAKE2b?

Reply:
This is present today

```
> crypto.getHashes().filter(x=>x.includes('blake'))
[ 'blake2b512', 'blake2s256' ]
```

Reply:
👍 excellent, thank you!

Reply:
```js
crypto.createHmac('blake2b512', 'panva').update('paseto').digest('hex')
// '045899e51fa51940d91448f3fa72626fd2bfbb25a64a9416d77a296ed962b0354ead6ea20c3c038c834b7d7c45add05c22bdd336758a0626a44c96a956ea4897'
```

Reply:
```js
> crypto.createHash('blake2b512').update('paseto').digest('hex')
// '1787f53c11b9db40f2580af58beb0e72b2e2c25072e1d4636b4c548c615840e32cbcf2e8fbd7a9a09564398fc768c5a111a472c4c5c579d0ccdc824b9fa09713'
```

Reply:
```js
Buffer.from(crypto.hkdfSync('blake2b512', 'key', 'salt', 'info', 32)).toString('hex')
// 'b3b27a54dac96f4819e8befbc643bd78b7882208b2e366043cbf47ab44b033a4'
```

Reply:
Does this let you use keyed BLAKE2b directly, or should I consider using HMAC instead?

Reply:
What you see above is the full extent of the exposed API today. `crypto.createMac` [work](https://github.com/nodejs/node/pull/32477) (which i think you're asking about?) did not bear fruit yet.